openslides_backend/permissions/__init__.py

```diff
@@ -0,0 +1,16 @@
+from typing import Optional
+
+from .permissions import permissions
+
+
+def is_child_permission(child: str, parent: str) -> bool:
```

Comment:
It seems to me that this algorithm in conjunction with permissions.py will not work correctly: A child can have more than one parent, that suffices the permission requirement, in the permission.yml. For instance to gain access for permission  `list_of_speakers.can_see` having permission `list_of_speakers.can_be_speaker
` is sufficient. `list_of_speakers.can_manage` instead should be, but isn't. It is not marked as `parent` of `list_of_speakers.can_see`.

Reply:
You are right! I did miss the double entry there. Well, this makes my nice algorithm a lot more complex...

Reply:
Fixed it. Thanks for the catch!